pom.xml

```diff
@@ -36,6 +36,7 @@
 
   <properties>
     <jackson.version>2.8.7</jackson.version>
+    <skip.tests>true</skip.tests>
```

Comment:
does `mvn test` still work?

Reply:
@efossier `mvn test -Dskip.tests="false"`

Reply:
Is it possible to turn it on by default but off for specifically releasing?